helpFile.md

```diff
@@ -0,0 +1,51 @@
+# Welcome to inorout
```

Comment:
Slack messages do not allow for headers like this

Reply:
I think rather than headers, it is just a way of parsing the file.  Could split the file up over each section and give the user a bit at a time.  I.e. "click here to read more"

Reply:
Then we should probably consider storing the data differently, like in a JSON object with `title`s that maybe are bolded or something. Paragraphs could be stored in an array within the object, perhaps.